core/src/main/resources/hudson/slaves/SlaveComputer/systemInfo.jelly

```diff
@@ -31,18 +31,22 @@ THE SOFTWARE.
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
   <l:layout title="${it.displayName} ${%System Information}" permissions="${it.EXTENDED_READ_AND_CONNECT}">
     <st:include page="sidepanel.jelly" />
+    <link rel="stylesheet" href="${resURL}/jsbundles/simple-page.css" type="text/css" />
 
     <l:main-panel>
 
-      <h1>
+      <h1>${%System Information}</h1>
+      <h2>
         <span class="icon-lg">
           <l:icon src="${it.icon}"/>
         </span>
-        ${it.caption} ${%System Information}
-        <j:if test="${!empty(it.node.nodeDescription)}">
-          <span style="font-size:smaller">(${it.node.nodeDescription})</span>
-        </j:if>
-      </h1>
+        ${it.caption}
+      </h2>
```

Comment:
Why this change? (other than extracting the nodeDescription into its own element)

Reply:
Because the node description is also used in this page. But here shall be the description not changed. From my side, I will remove the description here. It is more or less useless information on this page.

Reply:
To clarify, changing from h1 to h1 + h2.

Reply:
I will try to build my jenkins tomorrow again. and paste here screenshots before && after. 
The page System Information looks very strange when you use some description.
Maybe we shall really remove the part with the description from this site. It is useless here. What you think?

Reply:
>  Maybe we shall really remove the part with the description from this site. It is useless here. What you think?

Seems like a good idea, it doesn't work like that with anything else.